egs/wenetspeech/ASR/prepare.sh

```diff
@@ -5,6 +5,7 @@ set -eou pipefail
 nj=15
 stage=0
 stop_stage=100
+use_whole_text=True
```

Comment:
Where is it used and what is its meaning?

Reply:
Em...actually, I want to set a choice for using the transcripts in L subset or using the text in the whole data to generate the text, token.txt, words.txt and so on. But it seems that it is not finished. Will add some things about it.